src/main/java/mate/academy/hibernate/relations/service/impl/MovieServiceImpl.java

```diff
@@ -1,16 +1,25 @@
 package mate.academy.hibernate.relations.service.impl;
 
+import java.util.Optional;
+import mate.academy.hibernate.relations.dao.impl.MovieDaoImpl;
 import mate.academy.hibernate.relations.model.Movie;
 import mate.academy.hibernate.relations.service.MovieService;
 
 public class MovieServiceImpl implements MovieService {
+    private MovieDaoImpl movieDao;
```

Comment:
The type of the `movieDao` field should be the interface `MovieDao` instead of the implementation `MovieDaoImpl` to follow the Dependency Inversion Principle. This allows for more flexible code that is easier to test and maintain.